### ISSUE TITLE
Error with example

### ISSUE THREAD

**Issue by klofisch** [slimkrazy/python-google-places#44]
Hello,

when i run your example

```
from googleplaces import GooglePlaces, types, lang

YOUR_API_KEY = 'my_key'

google_places = GooglePlaces(YOUR_API_KEY)


query_result = google_places.nearby_search(
        location='London, England', keyword='Fish and Chips',
        radius=20000, types=[types.TYPE_FOOD])
```

i am getting this error:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 542, in _generate_lat_lng_string
    else geocode_location(location))
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 103, in geocode_location
    {'address': location, 'sensor': str(sensor).lower()})
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 71, in _fetch_remote_json
    request_url, response = _fetch_remote(service_url, params, use_http_post)
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 54, in _fetch_remote
    if type(v) in [str, unicode]:
NameError: name 'unicode' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/klofisch/PycharmProjects/Gmaps/Test01/test01.py", line 22, in <module>
    radius=20000, types=[types.TYPE_FOOD])
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 260, in nearby_search
    lat_lng_str = self._generate_lat_lng_string(lat_lng, location)
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 545, in _generate_lat_lng_string
    'lat_lng must be a dict with the keys, \'lat\' and \'lng\'')
ValueError: lat_lng must be a dict with the keys, 'lat' and 'lng'
```

**Comment by 5amfung** [comment]
I have the same issue.  Anyone has any idea how to fix this?

**Comment by slimkrazy** [comment]
I'm looking at thios now. You guys are using python3, right?

**Comment by ayasya** [comment]
I am facing the same prob.- " ValueError: lat_lng must be a dict with the keys, 'lat' and 'lng' ," since today.
I tried passing lat and lng as a dict with appropriate values and still the same problem.

**Comment by phixMe** [comment]
I am also seeing the second half of the reported issue:
Traceback (most recent call last):
  File "/home/klofisch/PycharmProjects/Gmaps/Test01/test01.py", line 22, in <module>
    radius=20000, types=[types.TYPE_FOOD])
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 260, in nearby_search
    lat_lng_str = self._generate_lat_lng_string(lat_lng, location)
  File "/usr/local/lib/python3.4/dist-packages/googleplaces/__init__.py", line 545, in _generate_lat_lng_string
    'lat_lng must be a dict with the keys, \'lat\' and \'lng\'')
ValueError: lat_lng must be a dict with the keys, 'lat' and 'lng'

However, this only happens on one of my two computers, my other computer with the same exact version of python works flawlessly. Not sure what to say...

**Comment by jordanjambazov** [comment]
The issue seems to be resulted by incompatibility with Python 3. I tried to execute it with Python 2.7 and it works fine.

The unicode function is not part of the standard library for Python 3, and one of the potential fixes is to use a `unicode = str` workaround for Python 3.

After fixing this, there also seems to be an issue with the urllib module not having the urlencode function in Python 3.

**Comment by lunoho** [comment]
Hello. There is already a pull request to fix py3 support: https://github.com/slimkrazy/python-google-places/pull/38

Please consider merging this pull request. Thanks!

**Comment by parthion** [comment]
I am facing the same problem, but I am running Python 2.7. 

Strangely enough, the script was executing just fine this past week, and suddenly started raising this error.

**Comment by phixMe** [comment]
I also originally  faced the problem in Python 2.7. I tried out Python 3.3 as well with the same exact error.

**Comment by chayanayak** [comment]
I am having the same problem on Python 2.7. The script was running fine and was actually in the middle of a large job when it started raising this error.  Has a solution been found yet?

**Comment by solidevolution** [comment]
Same problem here with python 3.4 ...

**Comment by slimkrazy** [comment]
This has been resolved by merging in #38.

The changes are currently on the development branch. I am attempting to address a few issues today before merging to master and releasing over the next day or so.

**Comment by slimkrazy** [comment]
Fixed and released in v1.1.0

**Comment by smyles140** [comment]
This error points to your radius=50000 when the '''ValueError: lat_lng must be a dict with the keys, 'lat' and 'lng' ''' error is given. Basically, this says that the given location doesn't have a given Latitude and Longitude for your specified search relating to that area and therefore cannot draw a radius around that area because it doesn't have a point (lat,lng) in the google database to start from. Make sense?
